dotcom-rendering/src/web/components/TopicFilterBank.tsx

```diff
@@ -0,0 +1,73 @@
+import { css } from '@emotion/react';
+import { from, headline, space, textSans } from '@guardian/source-foundations';
+import { decidePalette } from '../lib/decidePalette';
+import { FilterButton } from './FilterButton.importable';
+
+type Props = {
+	topics: Topic[];
+	format: ArticleFormat;
+};
+
+const containerStyles = css`
+	padding: ${space[3]}px 0;
+	width: 100%;
+`;
+const headlineStyles = css`
+	${headline.xxxsmall({ fontWeight: 'bold', lineHeight: 'tight' })}
+	padding-bottom: ${space[3]}px;
+`;
+
+const headlineAccentStyles = css`
+	${textSans.small({ fontWeight: 'regular', lineHeight: 'tight' })};
+`;
+
+const topicStyles = css`
+	display: flex;
+	align-items: flex-start;
+	flex-direction: row;
+	flex-wrap: wrap;
+	column-gap: ${space[2]}px;
+	row-gap: ${space[2]}px;
+	${from.desktop} {
+		flex-direction: column;
+		flex-wrap: nowrap;
+		column-gap: 0;
+	}
+`;
+
+export const TopicFilterBank = ({ topics, format }: Props) => {
+	const palette = decidePalette(format);
+	return (
+		<div css={containerStyles}>
+			<div css={headlineStyles}>
+				Filters{' '}
+				<span css={headlineAccentStyles}>
+					(
+					<span
+						css={css`
```

Comment:
For consistency does it make sense for this to be defined outside the component like the other styles?

Reply:
Yeah makes sense, I've refactored this.